data/en/test_radio_mandatory.json

```diff
@@ -0,0 +1,43 @@
+{
+    "mime_type": "application/json/ons/eq",
+    "schema_version": "0.0.1",
+    "data_version": "0.0.2",
+    "survey_id": "0",
+    "title": "Other input fields",
+    "theme": "default",
+    "legal_basis": "StatisticsOfTradeAct",
+    "groups": [{
+        "id": "radio",
+        "title": "RadioMandatory",
+        "blocks": [{
+                "type": "Questionnaire",
+                "id": "radio-mandatory",
+                "questions": [{
+                    "type": "General",
+                    "id": "radio-mandatory-question",
+                    "title": "Mandatory radio question, select an option",
+                    "answers": [{
+                        "type": "Radio",
+                        "id": "radio-mandatory-answer",
+                        "label": "What do you prefer for breakfast?",
```

Comment:
This is the question, so should be in the question title property. The label can be omitted. This applies to anywhere else this is done.